src/Symfony/Component/VarDumper/Tests/CliDumperTest.php

```diff
@@ -104,8 +104,182 @@ public function testGet()
 
 EOTXT
             ,
+            $out
+        );
+    }
+
+    /**
+     * @runInSeparateProcess
+     * @preserveGlobalState disabled
+     */
+    public function testSpecialVars56()
+    {
+        if (PHP_VERSION_ID < 50600) {
+            $this->markTestSkipped('PHP 5.6 or the symfony_debug extension is required.');
```

Comment:
should you check for the extension being loaded to avoid skipping in this case ?